### DISCUSSION TITLE
Store data with select with 1-N relationship

### DISCUSSION THREAD

**Discussion by dmatis2** [Laravel-Backpack/community-forum#489] [Q&A (Help)] [answered]
Hi, I have two models, interconnected with `hasMany` and `belongsTo` relationship. I'd like to store also selected value from that select. Now I have `null` there. How can I achieve that ?

**Comment by christmex** [comment, marked as answer]
Hi, can you please explain more? I'm sorry not that understand, but what I get is you want to show data from the relationship table with select right? maybe you can use this reference

https://backpackforlaravel.com/docs/5.x/crud-fields#select-1-n-relationship

Also, let me know if you have any issues, thx

**Reply by dmatis2** [reply]
I forgot to add foreign key property in migration, but thank you :) I'm closing it now
